sdk/storage/src/table/requests/delete_entity_builder.rs

```diff
@@ -64,7 +64,7 @@ impl<'a> DeleteEntityBuilder<'a> {
 
     pub fn to_transaction_operation<E>(
         &self,
-        entity: &E,
```

Comment:
Presumably we should just get rid of this argument completely no?

Reply:
I think so, I'm not sure why it was added in the first place.
I thought there might be some way that the api can be used where an entity might be needed, but the only relevant thing that I can find is the ETag, which isn't in the entity.
I'll remove it.